tests/utils.py

```diff
@@ -0,0 +1,263 @@
+import abc
+import itertools
+import unittest
+
+import numpy as np
+
+import dwave.optimization
+from dwave.optimization import (
+    Model,
+)
+
+
+class symbol_testing_utils:
```

Comment:
This was to create a "namespace" in the other file. Now that it's in its own I think we can move everything up a level.

Reply:
I tried that originally, got the same error, and so thought it still didn't work for some reason so I left it in the "namespace". But I just tried to remove the namespace and now it works. :shrug: